code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -732,6 +732,7 @@
 	has_corpse = TRUE
 	random_eligible = FALSE
 	info_text = "<span class='bold alloy'>As a </span><span class='bold brass'>Clockwork Golem Servant</span><span class='bold alloy'>, you are faster than other types of golems.</span>" //warcult golems leave a corpse
+	speedmod = 0 //yogs change
```

Comment:
unrequired, they already inherit speed 0  from /datum/species/golem/clockwork (this is /datum/species/golem/clockwork/no_scrap)

Reply:
check latest commit your view is outdated